### DISCUSSION TITLE
Warning: networking feature is disabled in Firejail configuration file

### DISCUSSION THREAD

**Discussion by YASOONA** [netblue30/firejail#6456] [Q&A] [answered]
Thanks for all the help so far. As told by @kmk3 I could open a new discussion, as mentioned [here ](https://github.com/netblue30/firejail/discussions/6453#discussioncomment-10506030):

    Warning: networking feature is disabled in Firejail configuration file

> It depends on what you're trying to do, but it's probably not a good idea to
> disable it unless you know what you're doing; feel free to open a discussion
> about it if you're unsure.

I haven't made any changes to the `restricted-network yes` in the [/etc/firejail/firejail.config](https://github.com/netblue30/firejail/blob/b9fe3cca6a0921d35d919f234e3f9faf7910ce3f/etc/firejail.config#L96). When I set the option to `restricted-network no`, obviously the error disappears. 

However rusty-snake [wrote](https://github.com/netblue30/firejail/issues/2888#issuecomment-518021048):

> `restricted-network` = yes` means that a normal use can't use `--interface, --eth` (except of `--net=none`) and `--netfilter`. If you need 
no one of this, you can set it to yes without issues (and harden your system a little bit more).

As I wrote in my previous discussion I'm new to Firejail.

Environment:

- Mx Linux 23.2 Minimal version (Debian)
- Firejail 0.9.72-2
- Firetools 0.9.72-1

Relates to:

* #7073

**Comment by rusty-snake** [comment]
So what's your TL;DR question?

Whether you should touch it?
If you need it (networking features), touch it. If you don't need it, don't touch it.

Why it is disabled by default?
Upstream it is enabled by default. Downstream (debian-only) it is disabled by default.

**Reply by YASOONA** [reply]
I'm not going to use any network-features at the moment. Maybe a printer in the future. So I can safely ignore this Warning? Thanks!

**Reply by rusty-snake** [reply, marked as answer]
> Maybe a printer in the future.

`noprinters` isn't gated by this.

> So I can safely ignore this Warning?

Yes.

**Comment by kmk3** [comment]
PR created to clarify this:

* #7074
